configurations/audit.yaml

```diff
@@ -1,5 +1,5 @@
-append_scylla_yaml: |
-  audit: "syslog"
-  audit_categories: "DCL,DDL,AUTH,ADMIN"
-  audit_tables: "keyspace1.standard1,scylla_bench.test"
-  audit_keyspaces: "keyspace1,scylla_bench"
+append_scylla_yaml:
+  audit: 'syslog'
```

Comment:
our of curiosity, when move from double quote to single ?

Reply:
Single quotes are always safer in yaml, so that there is no need to escape spec. characters (like `\` or `"`, etc.) if they are introduced to string values in future.

Reply:
I agree, e.g. double quotes in `defaults/manager_persistent_snapshots.yaml` caused us quite a lot of troubles - as are included in parsed dict.